Support/Multitenancy/Ironic_bmo-mult-instances/dowanload-node-image.sh

```diff
@@ -1,5 +1,5 @@
-cd /opt/metal3-dev-env/ironic/html/images/
+pushd /opt/metal3-dev-env/ironic/html/images/ || exit
```

Comment:
No need to have `|| exit` when using `pushd`.

Reply:
we need it, shellcheck was not happy in its absense

Reply:
if I remove it

In ./Support/Multitenancy/Ironic_bmo-mult-instances/dowanload-node-image.sh line 1:
pushd /opt/metal3-dev-env/ironic/html/images/
^-- SC2164 (warning): Use 'pushd ... || exit' or 'pushd ... || return' in case pushd fails.

Did you mean: 
pushd /opt/metal3-dev-env/ironic/html/images/ || exit

Reply:
Ok weird. I thought pushd would anyway give an exit code if it failed...

Reply:
```console
Exit Status:
    Returns success unless an invalid argument is supplied or the directory
    change fails.
```
It does return failure if it fails. But since the script does not have `set -e`, it requires `|| exit`.

Btw, just noticed the script name have funny typo: ´dowanload-node-image.sh` :)